spec/models/user_spec.rb

```diff
@@ -25,4 +25,33 @@
     it { is_expected.to delegate_method(:service_owner?).to(:organisation) }
     it { is_expected.to delegate_method(:delivery_partner?).to(:organisation) }
   end
+
+  describe "#email_validation" do
```

Comment:
Is this still a method name we're interested in now we're using the `EmailValidator`?